google-cloud-firestore/src/main/java/com/google/cloud/firestore/Query.java

```diff
@@ -1162,6 +1186,113 @@ public void onCompleted() {
         null);
   }
 
+  /**
+   * Returns the {@link RunQueryRequest} that this Query instance represents. The request contains
+   * the serialized form of all Query constraints.
+   *
+   * <p>Runtime metadata (as required for `limitToLast()` queries) is not serialized and as such,
+   * the serialized request will return the results in the original backend order.
+   *
+   * @return the serialized RunQueryRequest
+   */
+  public RunQueryRequest serializeToProto() {
```

Comment:
The internal convention seems to be `toProto` and `fromProto`, which seems perfectly fine. Why the extra long name here?

Reply:
No strong opinion here. Renamed.